packages/terra-date-time-picker/tests/wdio/date-time-picker-spec.js

```diff
@@ -1442,6 +1442,30 @@ Terra.describeViewports('DateTimePicker', ['large'], () => {
             expect($('input[name="terra-time-hour-input"]')).toHaveValue('02');
             expect($('input[name="terra-time-minute-input"]')).toHaveValue('00');
           });
+
+          it('should show the correct time when the day moves past the start date of  DST', () => {
+            browser.url('/raw/tests/cerner-terra-framework-docs/date-time-picker/date-time-picker-default-dst');
+            browser.refresh();
+            $('input[name="terra-date-year-input"]').click();
+            browser.keys('2020');
+            $('input[name="terra-date-month-input"]').click();
+            browser.keys('03');
+            $('input[name="terra-date-day-input"]').click();
+            browser.keys('07');
+            $('input[name="terra-time-hour-input"]').click();
+            browser.keys('02');
```

Comment:
```suggestion
            browser.keys('03');
```
Can we keep time as 3:30 and validate upon `+` key press validate the missing hour.

Reply:
Missing Hour is 2AM-3AM so any time with hour as 2 will change to 3 and then back to 2 for all other days.